packages/bot-skeleton/src/scratch/hooks/field_image.js

```diff
@@ -22,6 +23,7 @@ Blockly.FieldImage = function(src, width, height, optAlt, optOnClick, optFlipRtl
     this.tooltip_ = '';
     this.setValue(src);
     this.setText(optAlt);
+    this.isInFlyout = isInFlyout;
```

Comment:
Instead of passing an additional variable, you could check `this.sourceBlock_.isInFlyout`.

Reply:
I think it's more readable to pass this variable